src/app/pages/settings/comingSoon.js

```diff
@@ -1,115 +1,131 @@
-			);
+					'Coming soon page is not active. Site is live to visitors.',
+					'wp-plugin-bluehost'
+			  );
 	};
 
 	const toggleComingSoon = () => {
-		bluehostSettingsApiFetch({ comingSoon: !comingSoon }, setError, (response) => {
-			setComingSoon(!comingSoon);
-		});
+		bluehostSettingsApiFetch(
+			{ comingSoon: ! comingSoon },
+			setError,
+			// eslint-disable-next-line no-unused-vars
+			( response ) => {
+				setComingSoon( ! comingSoon );
+			}
+		);
 	};
 
 	const notifySuccess = () => {
-		notify.push("coming-soon-toggle-notice", {
+		notify.push( 'coming-soon-toggle-notice', {
 			title: getComingSoonNoticeTitle(),
-			description: (
-				<span>
-					{getComingSoonNoticeText()}
-				</span>
-			),
-			variant: "success",
+			description: <span>{ getComingSoonNoticeText() }</span>,
+			variant: 'success',
 			autoDismiss: 5000,
-		});
+		} );
 	};
 
-	useUpdateEffect(() => {
-		setStore({
+	useUpdateEffect( () => {
+		setStore( {
 			...store,
 			comingSoon,
-		});
+		} );
 
 		notifySuccess();
-		comingSoonAdminbarToggle(comingSoon);
-	}, [comingSoon]);
+		comingSoonAdminbarToggle( comingSoon );
+	}, [ comingSoon ] );
 
 	const getComingSoonSectionTitle = () => {
 		const getStatus = () => {
-			return (
-				comingSoon 
-				? <span className="nfd-text-[#e10001]">{__('Coming Soon', 'wp-plugin-bluehost')}</span>
-				: <span className="nfd-text-[#008112]">{__('Live', 'wp-plugin-bluehost')}</span>
+			return comingSoon ? (
+				<span className="nfd-text-[#e10001]">
+					{ __( 'Coming Soon', 'wp-plugin-bluehost' ) }
+				</span>
+			) : (
+				<span className="nfd-text-[#008112]">
+					{ __( 'Live', 'wp-plugin-bluehost' ) }
+				</span>
 			);
 		};
 
 		return (
-			<span>{__('Site Status', 'wp-plugin-bluehost')}: {getStatus()}</span>
-		)
+			<span>
+				{ __( 'Site Status', 'wp-plugin-bluehost' ) }: { getStatus() }
+			</span>
+		);
 	};
 
 	return (
 		<SectionSettings
-			title={getComingSoonSectionTitle()}
-			description={__('Still building your site? Need to make a big change?', 'wp-plugin-bluehost')}
+			title={ getComingSoonSectionTitle() }
+			description={ __(
+				'Still building your site? Need to make a big change?',
+				'wp-plugin-bluehost'
+			) }
 		>
 			<div className="nfd-flex nfd-flex-col nfd-gap-6">
 				<ToggleField
 					id="coming-soon-toggle"
 					label="Coming soon page"
-					description={__(
+					description={ __(
 						'Your Bluehost Coming Soon page lets you hide your site from visitors while you make the magic happen.',
 						'wp-plugin-bluehost'
-					)}
-					checked={comingSoon}
-					onChange={() => {
+					) }
+					checked={ comingSoon }
+					onChange={ () => {
 						toggleComingSoon();
-					}}
+					} }
 				/>
 
-				{comingSoon &&
+				{ comingSoon && (
 					<Alert variant="info">
-						{__('Your website is currently displaying a "Coming Soon" page.', 'wp-plugin-bluehost')}
+						{ __(
+							'Your website is currently displaying a "Coming Soon" page.',
```

Comment:
```suggestion
							'Your site is currently displaying a "Coming Soon" page.',
```

If we prefer "website", then there's a few other places we should update from "site" → "website" for consistency.